.github/workflows/php-test-workflow.yml

```diff
@@ -0,0 +1,78 @@
+name: Tests
+
+on:
+    push:
+    pull_request:
+
+jobs:
+    build:
+        name: Tests
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                include:
+                -   php: '7.4'
+                    dependencies-preference: " "
+                -   php: '7.4'
+                    dependencies-preference: "--prefer-lowest"
+                -   php: '8.0'
+                    dependencies-preference: " "
+                -   php: '8.0'
+                    dependencies-preference: "--prefer-lowest"
+#                -   php: '8.1'
+#                    dependencies-preference: " "
+#                -   php: '8.1'
+#                    dependencies-preference: "--prefer-lowest"
+            fail-fast: false
+        steps:
+        -   uses: actions/checkout@v2
+            with:
+                fetch-depth: 0
+        -   name: Setup PHP
+            uses: shivammathur/setup-php@v2
+            with:
+                coverage: "xdebug"
+                ini-values: date.timezone=Europe/Paris,memory_limit=-1,default_socket_timeout=10,session.gc_probability=0,apc.enable_cli=1,zend.assertions=1
```

Comment:
What is the reason to add date.timezone and why to lower down default_socket_timeout?